run_batch_of_slides.py

```diff
@@ -28,7 +36,8 @@ def build_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(description='Run Trident')
 
     # Generic arguments 
-    parser.add_argument('--gpu', type=int, default=0, help='GPU index to use for processing tasks.')
+    parser.add_argument('--gpu', '--gpus', dest='gpus', type=int, nargs='+', default=[0],
```

Comment:
Let's keep the `gpu` argument too, so as not to break anything. We can have both with a deprecation warning for the `gpu` one.